jobs/haproxy/templates/proxies_cidrs.txt.erb

```diff
@@ -0,0 +1,21 @@
+# generated from proxies_cidrs.txt.erb
+<%
+require "base64"
```

Comment:
these dependencies are not used. You based this file on logic that supports compressed files. Likely we can make this simpler?

A `join` would help, e..g `cidrs.join("\n")`.